runtime/src/configs/mod.rs

```diff
@@ -199,3 +199,8 @@ impl pallet_faucet::Config for Runtime {
     type LockPeriod = LockPeriod;
     type WeightInfo = pallet_faucet::weights::SubstrateWeight<Runtime>;
 }
+
+impl aura_session::Config for Runtime {
+    type SessionLength = ConstU32<MINUTES>;
```

Comment:
Let's make this easier to read:

E.g.:
https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/westend/src/governance/tracks.rs#L218

```suggestion
    type SessionLength = ConstU32<{ 1 * MINUTES }>;
```